src/ws_server.cu

```diff
@@ -257,6 +276,9 @@ class AprilTagHandler : public seasocks::WebSocket::Handler {
   void stop() { running_ = false; }
 
  private:
+  IntegerArraySender tagIDSender_{"tag_id"};
+  //BooleanValueSender isConnectedSender_{"ORIN_CONNECTED"};
```

Comment:
what is going in here?  can we delete it or do you want help hooking it up?

Reply:
fixed. I was going to implement this but realized this isn't the right PR for it. Will do it on a subsequent PR.